server/liveblog/blogs/blogs.py

```diff
@@ -83,21 +83,13 @@ class BlogsVersionsResource(ArchiveVersionsResource):
     """
 
     datasource = {
-        'source': 'archive' + '_versions',
-        'filter': {'type': 'composite'}
+        'source': 'archive' + '_versions'
     }
 
 
-class BlogsVersionsService(BaseService):
-    def get(self, req, lookup):
-        if req is None:
-            req = ParsedRequest()
-        return self.backend.get('archive_versions', req=req, lookup=lookup)
-
-
 class BlogsResource(ArchiveResource):
     datasource = {
-        'source': 'archive',
```

Comment:
How does it work for the source name when you don't explicit it ?
Isn't better to add `source: 'blogs'` like you did bellow ?